docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -1,175 +1,326 @@
-| `exporters`                            | no       | Specified which exporters to install. See [Configuring Exporters](#configuring-exporters).                                                                                                                                                                                                                                        | Installs deploytime exporter only.     |
+This is the list of options that can be applied to `exporters.instances` section.
 
-## Configuring Exporters Overview
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [app_name](#app_name) | yes | - |
+| [exporter_type](#exporter_type) | yes | - |
+| [env_from_secrets](#env_from_secrets) | no | - |
+| [env_from_configmaps](#env_from_configmaps) | no | - |
+| [extraEnv](#extraenv) | no | - |
+| [enabled](#enabled) | no | `true` |
+| [custom_certs](#custom_certs) | no | - |
+| [image_tag](#image_tag) | no | - |
+| [image_name](#image_name) | no | - |
+| [source_url](#source_url) | no | - |
+| [source_ref](#source_ref) | no | - |
 
-An _exporter_ is a data collection application that pulls data from various tools and platforms and exposes it such that it can be consumed by Pelorus dashboards. Each exporter gets deployed individually alongside the core Pelorus stack.
+### app_name
 
-There are currently three _exporter_ types which needs to be specified via `exporters.instances.exporter_type` value, those are `deploytime`, `failure` or `comittime`.
+- **Required:** yes
+- **Type:** string
 
+    Set the exporter name.
 
-Exporters can be deployed via a list of `exporters.instances` inside the `values.yaml` file that corresponds to the OpenShift ConfigMap configurations from the `charts/pelorus/configmaps/` directory. Some exporters also require secrets to be created when integrating with external tools and platforms. A sample exporter configuration may look like this:
+### exporter_type
 
-```yaml
-exporters:
-  instances:
-  - app_name: deploytime-exporter
-    exporter_type: deploytime
-    env_from_configmaps:
-    - pelorus-config
-    - deploytime-config
-```
+- **Required:** yes
+- **Type:** string
 
-Additionally, you may want to deploy a single exporter multiple times to gather data from different sources. For example, if you wanted to pull commit data from both GitHub and a private GitHub Enterprise instance, you would deploy two instances of the Commit Time Exporter.
+    Set the exporter type. One of `deploytime`, `committime`, `failure`.
 
-Each exporter additionally takes a unique set of environment variables to further configure its integrations and behavior. These can be set by using example ConfigMap object configurations similarly to the kubernetes secrets and listing them under `env_from_configmaps` or under `env_from_secrets` accordingly. As shown below.
+### env_from_secrets
```

Comment:
We don't have anywhere information how to use Secret from the file via:

`oc apply -f secret_file.yaml`

Sample files are in our repo:
https://github.com/konveyor/pelorus/tree/master/charts/secrets

Note those secrets have embedded namespace:
https://github.com/konveyor/pelorus/blob/master/charts/secrets/github_example.yaml#L8

We could have one without namespace or:
`oc apply -f secret_file.yaml -n pelorus`

Reply:
I will add this info

Reply:
should I update https://github.com/konveyor/pelorus/tree/master/charts/secrets comments? they are point to old URL

Reply:
Yeah, but it's not that important atm.

Reply:
https://github.com/konveyor/pelorus/issues/801